### PR TITLE
Fix backend tests to resolve the error `Error: Unable to create new Pad`

### PR DESCRIPTION
**Pull request by yacchin1205**
The [backend tests of etherpad-lite](https://github.com/ether/etherpad-lite/actions/runs/9175232874/job/25234195816?pr=6396#step:13:1876) was reporting the error below:

```
  11) /home/runner/work/etherpad-lite/etherpad-lite/node_modules/.pnpm/ep_image_upload@1.0.105_ep_etherpad-lite@src/node_modules/ep_image_upload/static/tests/backend/specs/exportHTML.js
       "before each" hook for "returns HTML with img HTML tags":
     Error: Unable to create new Pad
      at createPad (/home/runner/work/etherpad-lite/etherpad-lite/node_modules/.pnpm/ep_image_upload@1.0.105_ep_etherpad-lite@src/node_modules/ep_image_upload/static/tests/backend/specs/exportHTML.js:12:34)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async Context.<anonymous> (/home/runner/work/etherpad-lite/etherpad-lite/node_modules/.pnpm/ep_image_upload@1.0.105_ep_etherpad-lite@src/node_modules/ep_image_upload/static/tests/backend/specs/exportHTML.js:29:5)
```

The reason may be that the backend test is not compliant with the latest etherpad-lite test method. For example, [the latest test for ep_font_size](https://github.com/ether/ep_font_size/blob/a49f3ae90d66c27431db281403b296b8111345a0/static/tests/backend/specs/api/exportHTML.js) seems to give the token obtained from the utility function to the Authorization header.
So, I fixed the test by referring to the other test.

### CODE REVIEW

**Review by SamTV12345 — APPROVED** [review]

**Comment by SamTV12345** [comment]
I don't have access to this repo unfortunately and don't know if the maintainer is still around. Otherwise we could fork the repo and move it to ether.

**Comment by webzwo0i** [comment]
@tiblu 
Do you think it's okay to friendly fork ep_image_upload on our public ether account?
It is one of the most used plugins :-)

**Comment by tiblu** [comment]
@webzwo0i 

> @tiblu Do you think it's okay to friendly fork ep_image_upload on our public ether account? It is one of the most used plugins :-)

I am no longer part of CitizenOS team, that said I don't see an issue with that. BUT @ilmartyrk  is the right person to make a final decision on that.

**Pull request merged**
